File: PRAK205-2510817220020-LailyAzizahIslami.c
#include <stdio.h>
#include <math.h>

int main(){
    float A, B;
    float alas, tinggi, keliling, luas;

    scanf("%f %f", &A, &B);

    alas = sqrt(pow(B, 2) - pow(A, 2));
    tinggi = A;
    keliling = A + B + alas;
    luas = (alas * tinggi) / 2;

    printf("\nAlas = %.0f cm\n", alas);
    printf("Tinggi = %.0f cm\n", tinggi);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas = %.0f cm^2\n\n", luas);

return 0;
}

File: PRAK205-2510817220020-LailyAzizahIslami.py
import math

nilai = input().split()

if len(nilai) == 1:
    a = float(nilai[0])
    b = float(input())
else:
    a, b = map(float, nilai)

alas = (pow(b, 2) - pow(a, 2)) ** 0.5
tinggi = a
keliling = a + b + alas
luas = 0.5 * alas * a

print(f"\nAlas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")